packages/vue-component-type-helpers/package.json

```diff
@@ -10,5 +10,8 @@
 		"type": "git",
 		"url": "https://github.com/vuejs/language-tools.git",
 		"directory": "packages/vue-component-type-helpers"
+	},
+	"dependencies": {
+		"@vue/reactivity": "^3.3.0"
```

Comment:
`vue-component-type-helpers` should not depend on `@vue/reactivity`.

Reply:
Thank you! But how could I ShallowUnwrapRef the exposed?

Reply:
> `vue-component-type-helpers` should not depend on `@vue/reactivity`.

Sorry I don't knew that. Thank you very much for your PR so soon.